### PR TITLE
[IMP] account: resequence only in debug mode

### PR DESCRIPTION
**Pull request by william-andre**
Calm the concerns of quite a few users who, rightly or wrongly, consider
the resequencing wizard as a blocking point regarding de sales journal.

[task-2604155](https://www.odoo.com/web#cids=1&id=2604155&model=project.task)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/models/ir_actions.py [hunk 1]
```diff
@@ -82,28 +82,31 @@ def _get_eval_context(self, action=None):
...
+        if not debug:
+            user_groups -= self.env.ref('base.group_no_one')
+
+        self.flush()
+        cr.execute("""
+            SELECT a.id, a.type, a.binding_type
+              FROM ir_actions a
+              JOIN ir_model m ON a.binding_model_id = m.id
+             WHERE m.model = %s
+          ORDER BY a.id
```
Comment (rco-odoo) [inline odoo/addons/base/models/ir_actions.py:108]: the random indentation in this query is disturbing to me

Comment (william-andre) [reply odoo/addons/base/models/ir_actions.py:112]: @KangOl Round 1. Ready?... Fight!

File: odoo/addons/base/models/ir_actions.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from odoo.osv import expression
 from odoo.tools.safe_eval import safe_eval, test_python_expr
 from odoo.tools.float_utils import float_compare
+from odoo.http import request
```
Comment (rco-odoo) [inline odoo/addons/base/models/ir_actions.py:10]: :+1: this import looks better than adding one in `odoo/models.py`

Comment (william-andre) [reply odoo/addons/base/models/ir_actions.py:10]: https://github.com/odoo/odoo/blob/1c89dd90937c8b313397c4d91ee1f1006686d01e/odoo/models.py#L1459

:slightly_smiling_face:

Comment (rco-odoo) [reply odoo/addons/base/models/ir_actions.py:10]: The purpose was to explicitly avoid import cycles at the file level.

**Review by rco-odoo — APPROVED** [review]

File: addons/account/wizard/account_resequence_views.xml [hunk 3]
```diff
@@ -26,14 +26,15 @@
             </form>
             </field>
         </record>
-
+    </data>
+    <data noupdate="1">
```
Comment (KangOl) [inline addons/account/wizard/account_resequence_views.xml:30]: The record now being in noupdate, an upgrade script is needed to force the update.

Comment (william-andre) [reply addons/account/wizard/account_resequence_views.xml:30]: Done https://github.com/odoo/upgrade/pull/2794

**Review by oco-odoo — APPROVED** [review]
@robodoo r+ rebase-ff

**Pull request closed without merging**